Specify the return value (#29942)
mdn/content: files/en-us/web/api/urlsearchparams/getall/index.md

### Return value

> An array of strings or an empty array if parameter is not found.

Reviewer: ```suggestion
An array of strings, which may empty if no values for the given parameter are found.
```
